### PR TITLE
[16.2.x] build: lock file maintenance

### PR DESCRIPTION
**Pull request by josephperrott**
Update the lock file.

### CODE REVIEW

**Comment by JeanMeche** [comment]
See #51834, the new symbols are related to the terser update https://github.com/terser/terser/commit/33d53e0cf24fffeb935f720e1f84cc66b9c8bc3a.

**Review by alan-agius4 — DISMISSED** [review]

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -3,6 +3,7 @@ name: CI
 on:
   push:
     branches:
+      - 'update-lockfile-patch'
```
Comment (devversion) [inline .github/workflows/ci.yml:6]: This should be reverted right?

**Review by devversion — DISMISSED** [review]
LGTM

Reviewed-for: global-dev-infra-approvers

**Review by alxhub — APPROVED** [review]
Reviewed-for: global-approvers

**Comment by alxhub** [comment]
This PR was merged into the repository by commit 781bb91f436cdb3aeb983406208ac2bda6ce8813.

**Pull request closed without merging**